src/blueapi/service/authentication.py

```diff
@@ -54,14 +56,16 @@ def load_cache(self) -> Cache:
     def delete_cache(self) -> None:
         Path(self._file_path).unlink(missing_ok=True)
 
-    def _get_xdg_cache_dir(self) -> Path:
+    def _get_xdg_cache_path(self) -> Path:
```

Comment:
```suggestion
    def _default_token_cache_path(self) -> Path:
```
This doesn't always return the xdg_cache. See also the docstring